src/InteractsWithBanner.php

```diff
@@ -12,7 +12,7 @@ trait InteractsWithBanner
      */
     protected function banner($message)
     {
-        $this->dispatchBrowserEvent('banner-message', [
+        $this->dispatch('banner-message', [
```

Comment:
When dispatching from the backend, instead of passing an array, you should pass named arguments:

```php
$this->dispatch('banner-message', 
    style: 'success',
    message: $message,
);
```

And this would remove the later `$event.detail[0].style` syntax in this PR. It would become simply: `$event.detail.style` (just like before)

I'll make a note on that too